tests/node_service/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "jfharden-metrics-experiment",
+  "version": "1.0.0",
+  "description": "Test service for testing metrics exporting",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.2",
+    "prom-client": "^14.2.0"
+  }
+}
```

Comment:
Can we add config for these npm packages into `dependabot.yml`? (The usual `open-pull-requests-limit: 0`)

Reply:
Oh yeah, totally forgot about the npm deps (I wasn't going to have any until the otel config validator didn't work :()

Reply:
Added in https://github.com/alphagov/pay-adot/pull/2/commits/094eabd0a24a92de11284461bf7353b82c4a13ab